bindings/py/cpp_src/bindings/algorithms/py_SDRClassifier.cpp

```diff
@@ -35,12 +35,14 @@ PyBind11 bindings for SDRClassifier class
 
 #include <nupic/algorithms/SDRClassifier.hpp>
 
+
+namespace nupic_ext
+{
 namespace py = pybind11;
 using namespace nupic;
 using namespace nupic::algorithms::sdr_classifier;
+using T = std::vector<Real64>;
```

Comment:
Nitpick: can this import the symbol `T` from the header file?

Reply:
I had hoped so, but it didn;t. I'll investigate if it works.